platformio.ini

```diff
@@ -26,6 +26,7 @@ lib_archive = false
 lib_deps = 
   androbi/MqttLogger
   bodmer/TFT_eSPI
+	knolleary/PubSubClient
```

Comment:
Whitespace ;)

Reply:
good to have you watching me ;-) 

done!